README.md

```diff
@@ -12,47 +12,45 @@
 [link-nvm]: https://github.com/creationix/nvm
 [link-nvm-installation-instructions]: https://github.com/creationix/nvm#installation
 
-The `docs-ui` repository is a custom version of the [Antora Default UI][link-Antora-Default-UI], 
-for the [Antora][link-Antora] version of the ownCloud documentation.
- 
-## Use the ownCloud Antora-based Docs Default UI
+The `docs-ui` repository is a custom version of the [Antora Default UI][link-Antora-Default-UI], for the [Antora][link-Antora] version of the ownCloud documentation.
 
-Any merged changes will automatically trigger the creation of a new `ui-bundle.zip` package file.
-This [ui_bundle.zip][link-ui_bundle.zip] file is automatically published and can be downloaded or used by scripts.
+## Contributing
 
-It contains the content that is valid for the whole documentation like `header`, `footer` ect. and is embedded during the documentation build process automatically.
+If you want to make changes, create a "_feature_" branch off of master, make the required changes, and then create a Pull Request (PR) against the _master_ branch.
+If the PR is accepted and merged, a new `ui-bundle.zip` package file will be created and published to https://minio.owncloud.com/documentation/ui-bundle.zip.
 
-If you want to see the result of unmerged UI changes, you need to manually create a `ui-bundle.zip` package.
+## Previewing Changes Locally
 
-## Package Creation for Use with Antora
-
-If you need to package the UI in order to preview unmerged changes in the UI in local development, 
-run the following command in the local root of your `docs-ui` clone:
+To view your changes as you are working on them, run the following command:
 
 ```
- $ gulp pack
+ $ gulp preview
```

Comment:
question, does gulp preview show the content without it has been packed first ?

Reply:
`gulp preview` generates a demo version of the site.

Reply:
It doesn't generate a ui bundle archive.